WordPress/gutenberg
# Mobile - Global styles: Set theme's background and text color

This PR is a continuation of https://github.com/WordPress/gutenberg/pull/30684

## Description
This PR adds support to render the theme's main background, text, and link color.

## How has this been tested?
Using the WordPress app build from these branches:
`WordPress-Android` PR: https://github.com/wordpress-mobile/WordPress-Android/pull/14597
`WordPress-iOS` PR: https://github.com/wordpress-mobile/WordPress-iOS/pull/16411

### Test case 1

- Open the WordPress app with metro running (from this PR)
- Create a new post
- Expect to see the theme's default background color (Green) and the text color (Dark gray)

### Text Case 2

- Open the WordPress app with metro running (from this PR)
- Create a new post
- Add a link
- Expect the color to be dark gray

### Text Case 3

- Open the WordPress app with metro running (from this PR)
- Create a new post
- Add a Paragraph block
- Open the blocks settings
- Expect the color option to be available with the theme's colors

## Screenshots

iOS | Android
-|-
<kbd><img src="https://user-images.githubusercontent.com/4885740/119994414-def89a00-bfcc-11eb-8f2b-3ff8f65ca06e.png" width="250" /></kbd> | <kbd><img src="https://user-images.githubusercontent.com/4885740/119994245-b53f7300-bfcc-11eb-8ac6-31c97d2feb0b.png" width="250" /></kbd>

## Types of changes
New feature

## Checklist:
